<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegmentationToolboxClassifyingWidget</class>
 <widget class="QWidget" name="SegmentationToolboxClassifyingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Learning</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="ImageRangePreprocessingSelector" name="imageRangePreprocessingSelector" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="addImages">
        <property name="text">
         <string>Add more parametric images</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addClassifier">
     <property name="text">
      <string>Add classifier</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Classifier</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="combination"/>
   </item>
   <item>
    <widget class="QPushButton" name="classify">
     <property name="text">
      <string>Classify</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageRangePreprocessingSelector</class>
   <extends>QWidget</extends>
   <header>ImageRangePreprocessingSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
